pkg/rpcserver/runner.go

```diff
@@ -203,6 +202,95 @@ func (runner *Runner) ConnectionLoop() error {
 	}
 }
 
+func (runner *Runner) recvExecutorMessage() (msg any, err0 error) {
```

Comment:
Maybe just make it a part of `flatrpc.Recv`? The package knows about all messages anyway.

Reply:
Done. PTAL.

The fuzz test can reproduce the OOM with the seed and the change fixes it.